locales/en.default.schema.json

```
@@ -1801,6 +1870,37 @@
             "label": "Large"
           },
           "info": "For best results, use an image with a 16:9 aspect ratio. [Learn more](https://help.shopify.com/en/manual/shopify-admin/productivity-tools/image-editor#understanding-image-aspect-ratio)"
+        },
+        "desktop_content_alignment": {
+          "options__1": {
+            "label": "Left"
+          },
+          "options__2": {
+            "label": "Center"
+          },
+          "options__3": {
+            "label": "Right"
+          },
+          "label": "Desktop content alignment"
+        },
+        "header": {
+          "content": "Mobile Layout"
+        },
+        "mobile_content_alignment": {       
+          "options__1": {
+            "label": "Left"
+          },
+          "options__2": {
+            "label": "Center"
+          },
+          "options__3": {
+            "label": "Right"
+          },
+          "label": "Mobile content alignment"
+        },
+        "show_text_below": {
+          "label": "Show content below image on mobile",
```

```suggestion
          "label": "Show content below image",
```
We shouldn't repeat "on mobile" under the "Mobile layout" header, at least at the end of this string of text. cc. @katycobb for visibility and a [question/reference](https://screenshot.click/21-11-jx8lk-1hdy9.png)